crates/graphql/src/schema.rs

```diff
@@ -51,6 +51,43 @@ impl<'a> From<models::Metadata<'a>> for Nft {
     }
 }
 
+#[derive(GraphQLObject)]
+#[graphql(description = "A Metaplex storefront")]
+pub struct Storefront {
+    pub owner_address: String,
+    pub subdomain: String,
+    pub title: String,
+    pub description: String,
+    pub favicon_url: String,
+    pub logo_url: String,
+    pub banner_url: String,
+}
+
+impl<'a> From<models::Storefront<'a>> for Storefront {
+    fn from(
+        models::Storefront {
+            owner_address,
+            subdomain,
+            title,
+            description,
+            favicon_url,
+            logo_url,
+            banner_url,
+            ..
+        }: models::Storefront,
+    ) -> Self {
+        Self {
+            owner_address: owner_address.into_owned(),
+            subdomain: subdomain.into_owned(),
+            title: title.into_owned(),
+            description: description.into_owned(),
+            favicon_url: favicon_url.into_owned(),
+            logo_url: logo_url.into_owned(),
+            banner_url: banner_url.unwrap().into_owned(),
```

Comment:
What is a better why to handle this? The code defaults to saving the records with "" but can be none.

Reply:
Does GraphQL support `banner_url` being an `Option<String>`?  If so it would simplify the code to something like `banner_url.map(Cow::into_owned)`